int32[] ids
---
spatial_db_msgs/ObjectDescription[] descriptions
